server/server.js

```diff
@@ -1,15 +1,76 @@
 const express = require("express");
 const app = express();
+const bodyParser = require("body-parser");
 const port = process.env.PORT || 5000;
-
 app.listen(port, () => console.log(`Listening on port ${port}`));
 
+app.use(bodyParser.urlencoded({ extended: false }));
+// parse application/json
+app.use(bodyParser.json());
 // Store and retrieve your videos from here
 // If you want, you can copy "exampleresponse.json" into here to have some data to work with
-let videos = [];
+let videos = require("./videos.json");
 
 // GET "/"
 app.get("/", (req, res) => {
-  // Delete this line after you've confirmed your server is running
-  res.send({ express: "Your Backend Service is Running" });
+  res.json(videos);
+});
+
+app.post("/", (req, res) => {
+  const newId = videos.length + 1;
```

Comment:
Are you sure this will always be a unique ID? Can you think of a time when this technique would end up using the same ID for two different videos?